### DISCUSSION TITLE
X-AnyLabeling v2.4.0

### DISCUSSION THREAD

**Discussion by CVHub520** [CVHub520/X-AnyLabeling#521] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/CVHub520/X-AnyLabeling/releases/tag/v2.4.0'>X-AnyLabeling v2.4.0</a>.</em>

**Comment by liuxiaohe123** [comment]
X-AnyLabeling-CPU.exe: error: argument command: invalid choice: 'C:\\Users\\LXH\\xanylabeling_data/trainer/ultralytics\\runs\\detect\\train_script.py' (choose from help, checks, version, config, convert)
ERROR: Training process exited with code 2

**Comment by CVHub520** [comment]
To use the Ultralytics training feature in X-AnyLabeling, please run the application from source code.

**Comment by liuxiaohe123** [comment]
OK thank you



---Original---
From: ***@***.***&gt;
Date: Tue, Dec 9, 2025 15:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [CVHub520/X-AnyLabeling] X-AnyLabeling v2.4.0 (Discussion #521)




 
To use the Ultralytics training feature in X-AnyLabeling, please run the application from source code.
